### PR TITLE
Release 2.9

### PR DESCRIPTION
**Pull request by Tetrergeru**

### CODE REVIEW

File: source/installation/configuration.rst [hunk 1]
```diff
@@ -57,6 +57,18 @@ reduce CPU usage on your server. Lazy triggers checker works if
 ``lazy_triggers_check_interval`` > ``check_interval``. We recommend set
 it to ``10m`` (10 minutes).
 
+.. _prometheus-remote-triggers-checker:
+
+Prometheus Checker
+^^^^^^^^^^^^^^^^^^
+
+In Moira 2.9 Prometheus Remote metric source was added. It works like Graphite
+remote metric source, but uses mrometheus metrics and PromQL instead. It makes
```
Comment (almostinf) [inline source/installation/configuration.rst]: > mrometheus

typo

File: source/installation/configuration.rst [hunk 2]
```diff
@@ -57,6 +57,18 @@ reduce CPU usage on your server. Lazy triggers checker works if
...
 it to ``10m`` (10 minutes).
 
+.. _prometheus-remote-triggers-checker:
+
+Prometheus Checker
+^^^^^^^^^^^^^^^^^^
+
+In Moira 2.9 Prometheus Remote metric source was added. It works like Graphite
+remote metric source, but uses mrometheus metrics and PromQL instead. It makes 
+queries to Prometheus api ar Victoria Metrics VMSelect api.
```
Comment (almostinf) [inline source/installation/configuration.rst]: > to Prometheus api ar Victoria Metrics

also typo

File: source/installation/configuration.rst [hunk 3]
```diff
@@ -57,6 +57,18 @@ reduce CPU usage on your server. Lazy triggers checker works if
...
+
+Proemtheus Checker can be counfigured to use retries when trying to fetch metrics.
```
Comment (almostinf) [inline source/installation/configuration.rst]: > Proemtheus Checker

and there

File: source/installation/configuration.rst (hunk 1 above)
Comment (Tetrergeru) [reply source/installation/configuration.rst]: Fixed

File: source/installation/configuration.rst (hunk 2 above)
Comment (Tetrergeru) [reply source/installation/configuration.rst]: Fixed

File: source/installation/configuration.rst (hunk 3 above)
Comment (Tetrergeru) [reply source/installation/configuration.rst]: Fixed

**Pull request closed without merging**